test.js

```diff
@@ -1,16 +1,19 @@
 // NOTE: to run this you will probably need to be authorized with GitHub.
 // Run `./changelog-maker.js` by itself to set this up.
 
-const test = require('tape')
+'use strict'
+
+const path = require('path')
 const execSync = require('child_process').execSync
+const test = require('tape')
 
 function exec (args) {
-  let stdout = execSync(`${process.execPath} ${__dirname}/changelog-maker.js ${args}`).toString()
+  const stdout = execSync(`"${process.execPath}" ${path.join(__dirname, 'changelog-maker.js')} ${args}`).toString()
```

Comment:
Wrapped `process.execPath` in double quotes because this fails on Windows due to the space in the path